examples/selalib_distributed/README.md

```diff
@@ -1,32 +1,54 @@
-## Selalib Example
+# Selalib Example
 
 * runs the Semi-Lagrangian BSL code
-* public part available here https://github.com/selalib/selalib
-* "our" currently used branch is https://gitlab.mpcdf.mpg.de/clapp/selalib/-/tree/exahd_communicator
+* public part available here <https://github.com/selalib/selalib>
+* "our" currently used branch is <https://gitlab.mpcdf.mpg.de/clapp/selalib/-/tree/exahd_communicator>
```

Comment:
I no longer have access to the MPCDF GitHub for SeLaLib. Does this branch need to be private? If not then would it be possible to put it in the GitHub or on a fork of the GitHub? That way the example can be useful for people who are not familiar with SeLaLib (or like me people who are familiar with SeLaLib but no longer have access to the newest developments)

Reply:
For general functionality testing, everything should be sufficiently public ([this interface](https://github.com/selalib/selalib/blob/main/simulations/parallel/bsl_vp_3d3v_cart_dd/sll_m_sim_bsl_vp_3d3v_cart_dd_slim_interface.F90) is the same in both repos). The numerical results we were getting were different due to various of @kkormann's edits as part of [this work](https://journals.sagepub.com/doi/10.1177/1094342019834644), so the public repo cannot be used to reproduce numerical results.

IIRC there was still something missing when running `spack install discotec+selalib` -- let me check.

Reply:
@EmilyBourne running `spack dev-build -b install discotec@main +selalib` in the DisCoTec folder built everything smoothly, you should see some `selalib_*` executables in `examples/selalib`! It links to the public version.

Reply:
correction: one of the postprocessing executables does not build correctly, will push a fix

Reply:
@EmilyBourne this here https://github.com/SGpp/DisCoTec/commit/38e8396df8a1ffa427afea680ff7cf1e5a4cb815 should make `discotec+selalib` work (of course with the usual spack dependency hell that kept me from putting it as a check into Jenkins ;) )